core/inject.js

```diff
@@ -387,7 +384,7 @@ Blockly.init_ = function(mainWorkspace) {
             mainWorkspace, true, true, 'blocklyMainWorkspaceScrollbar');
     mainWorkspace.scrollbar.resize();
   } else {
-    mainWorkspace.setMetrics({x: 0.5, y: 0.5});
+    mainWorkspace.setMetrics({x: 0, y: 0});
```

Comment:
Just curious, why the change?

Reply:
I believe I changed it because it fixed unit tests. However it looks like it doesn't make a difference now, so I'll change it back.
I think it's possible that the scrollbar logic refactor PR may have something to do with it.